specs/wp-page-editor-spec.js

```diff
@@ -281,7 +281,6 @@ test.describe( `[${host}] Editor: Pages (${screenSize})`, function() {
 				this.editorPage.enterTitle( pageTitle );
 				this.postEditorSidebarComponent = new PostEditorSidebarComponent( driver );
 				this.postEditorSidebarComponent.setVisibilityToPasswordProtected( postPassword );
-				this.editorPage = new EditorPage( driver );
```

Comment:
I think this is actually needed for mobile as it switches context between the sidebar screen and the editor screen